man/tpm2_startauthsession.1.md

```diff
@@ -0,0 +1,71 @@
+% tpm2_startauthsession(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_startauthsession**(1) - Start a session with the TPM.
+
+# SYNOPSIS
+
+**tpm2_startauthsession** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_startauthsession**(1) Starts a session with the TPM. The default is
+to start a *trial* session unless the **-a** option is specified. It outputs
+the session handle in a yaml format to stdout as key value "session-handle"
+and the handle in hex (0x1234) format.
+
+# OPTIONS
+
+  * **-a**, **--auth-policy-session**:
+
+    Change the policy session from a *trial* session to a *policy* session.
+    **NOTE**: A *trial* session is used when building a policy and a *policy*
+    session is used when authenticating with a policy.
+
+  * **-g**, **--policy-digest-alg**=_HASH\_ALGORITHM_:
+
+    The hash algorithm used in computation of the policy digest. Algorithms
+    should follow the "formatting standards", see section "Algorithm Specifiers".
+    Also, see section "Supported Hash Algorithms" for a list of supported hash
+    algorithms.
+
+  * **-S**, **--session**=_SESSION_FILE_:
+
+    Saves the policy session data to a file. This file can then be used in subsequent
+    tools that can use a policy file for authorization or policy events. **NOTE**: That
+    this will not work with resource managers (RMs) outside of abrmd, as most RMs will
+    flush session handles when a client disconects on the IPC channel. This will work
```

Comment:
shouldn't this be "from" the IPC channel?

Reply:
That and disconects should be spelled correctly, "disconnects".